src/generate/api.ts

```diff
@@ -12,7 +12,7 @@ import {
   getAllDataTypes
 } from "../common/parser";
 
-import Name from "./name";
+import { Name } from "./name";
```

Comment:
Doing a full review, noticed that `model` in the constructor is missing a type. Also, class definition can be simplified by using `public` in the constructor parameter list.